fontforge_font_creator/creator.py

```diff
@@ -36,8 +46,12 @@ def create_font(self):
             self.fontforge_font.weight = 'normal'
             for hex_position, glyph in definition['mappings'].items():
                 hex_value = int(hex_position, 16)
-                print(hex_value, glyph['filename'])
-                self.add_glyph(hex_value, glyph['filename'])
+                print(hex_value, glyph['filename'], glyph['element'])
```

Comment:
just progress feedback & debugging help

It as already there, I just extended it with another value. If you prefer, we can remove it.